src/axl_mpi.c

```diff
@@ -147,6 +150,7 @@ static int axl_create_dirs(int count, const char** filelist, MPI_Comm comm)
 
     return AXL_SUCCESS;
 }
+#endif
```

Comment:
why comment it out but keep it? Is it expected to be only temporarily not needed?

Reply:
Fixed in latest push